metrics/prometheus/prometheus.go

```diff
@@ -27,6 +27,7 @@ type Metrics struct {
 	impressionsLegacy            prometheus.Counter
 	prebidCacheWriteTimer        *prometheus.HistogramVec
 	requests                     *prometheus.CounterVec
+	debugRequests                prometheus.Counter
```

Comment:
Hmm.. why are the types for `debugRequests` and `accountDebugRequests` different? One is a `Counter` and other is a `CounterVec`

Reply:
Discussed offline. In Prometheus account metrics have an `account` label and so they need to be `CounterVec` than a plain `Counter`